scripts/package-distribution.ps1

```diff
@@ -122,6 +122,12 @@ foreach ($Build in $AllBuilds) {
         Copy-Item -LiteralPath $Library -Destination $CopyToFolder -Force
     }
 
+    # Copy License
+    Copy-Item -Path (Join-Path $RootDir "LICENSE.md") -Destination $TempDir
```

Comment:
We do not use the MD extension for the license file. We should also probably be consistent for third party notices as well.